content/md/first-steps/manage-asset-families.md

```diff
@@ -137,15 +142,15 @@ To define the options of a single/multiple options attribute, follow these steps
 1.  Click on the `Manage options` button
 1.  A fullscreen opens to manage the options
 
-![image](../img/AssetFamily_ManageOptions.png)
+![Manage options](../img/Assets_ManageOptions.png)
```

Comment:
C'est pas ici, mais y a une erreur sur le lien "manage the options of the list" dans "EDIT A SINGLE/MULTIPLE OPTIONS ATTRIBUTE"

Reply:
done